test/routes/entry-point.test.ts

```diff
@@ -23,9 +23,21 @@ describe('entry-point', (): void => {
     const { dataset } = await makeRequest();
     const id = namedNode('http://example.com/path-to/entry-point');
 
-    expect(dataset).toBeRdfDatasetContaining(quad(id, rdf.type, schema.EntryPoint));
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: schema('name') });
-    expect(dataset).toBeRdfDatasetMatching({ subject: id, predicate: hydra.collection });
+    expect(dataset).toBeRdfDatasetContaining(
+      quad(id, rdf.type, schema.EntryPoint),
+      quad(id, schema('name'), literal('Article Store', 'en')),
```

Comment:
surprising how language is a first class concept in the `literal`

Reply:
https://www.w3.org/TR/rdf11-concepts/#dfn-language-tagged-string